pkg/test/context.go

```diff
@@ -76,6 +78,14 @@ func (ctx *TestCtx) GetID() string {
 }
 
 func (ctx *TestCtx) Cleanup() {
+	if ctx.t != nil {
+		// The cleanup function will be skipped
+		if ctx.t.Failed() && ctx.skipCleanupOnError {
+			// Also, could we log the error here?
+			log.Info("Skipping cleanup function since --skip-cleanup-error is true and an error was faced")
```

Comment:
maybe just "Skipping cleanup function since --skip-cleanup-error is true"?

Reply:
But the skip just occurs in the failures scenarios. Could you help us to re-phrase it better?

Reply:
I was thinking about @jmccormick2001 suggestion here, wdyt @JAORMX of:
```suggestion
			log.Info("Skipping cleanup function since --skip-cleanup-error is true")
```

Reply:
Shows solved.